client/renderers/EGL/help.c

```diff
@@ -37,11 +38,12 @@ struct EGL_Help
   const LG_Font * font;
   LG_FontObj      fontObj;
 
-  EGL_Texture   * texture;
-  EGL_Shader    * shader;
-  EGL_Shader    * shaderBG;
-  EGL_Model     * model;
-  LG_FontBitmap * bmp;
+  EGL_Texture * texture;
+  EGL_Shader  * shader;
+  EGL_Shader  * shaderBG;
+  EGL_Model   * model;
+
+  LG_FontBitmap * volatile _Atomic bmp;
```

Comment:
`volatile` is not correct here, please use `_Atomic(LG_FontBitmap *) bmp;`
The atomic intrinsics handle cache coherency.

Reply:
Fixed.